### ISSUE TITLE
Suggestion: Enable asset swapping runtime.

### ISSUE THREAD

**Issue by TheWalkingFridge** [rive-app/rive-flutter#43]
This tool would be really useful. For instance, if a character is customizable it will allow using a single bone animation with the image asset (the one attached to the bone) chosen/swapped runtime.

**Comment by mjohnsullivan** [comment]
Thanks for the suggestion; we'll add this to our ideas list.

At the moment we don't have support for raster images in Rive 2 or the runtime, but that's planned for the future.

**Comment by TheWalkingFridge** [comment]
@mjohnsullivan is there a workaround for this now? I'm only looking to swap the animated SVG with another during runtime (no raster images here)

**Comment by mjohnsullivan** [comment]
We convert SVG images to Rive's format when adding to an animation; the animation itself doesn't currently have the concept of assets. Currently you'll need to add all the SVGs to the animation and handle swapping them at runtime.

Assets support is on our todo list, but I don't have a timeline for that yet.

**Comment by natebot13** [comment]
Now that asset support is available, could this feature request be brought back into attention? I would love to have this

**Comment by savire** [comment]
Yes please we need this feature for game characters so that we are able to swap it live with for cosmetic changes

**Comment by HayesGordon** [comment]
It is now possible to swap runtime assets, such as fonts, images, and audio: https://rive.app/community/doc/loading-assets/doct4wVHGPgC

The more dynamic swapping will fall part of larger feature set that will be part of the full Rive Editor experience. And it's not directly tied to the Flutter runtime. Closing this issue as we're keeping track of those features in a separate place
